### ISSUE TITLE
Show members people waiting for verification from them

### ISSUE THREAD

**Issue by markulrich** [rahafoundation/raha-mobile-app#316]
In short term, it seems like a natural place to put this would be in the `ReferralBonusPage`? Maybe use  a `SectionList` containing 1) people you need to verify and 2) completed invites you can Mint?

**Comment by rahulgi** [comment]
Good idea. We were already doing this when it was just a trust action required, and we should def re-add this feature.
